data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/parser/LogstashVisitor.java

```diff
@@ -0,0 +1,148 @@
+package org.opensearch.dataprepper.logstash.parser;
+
+import org.antlr.v4.runtime.RuleContext;
+import org.opensearch.dataprepper.logstash.exception.LogstashParsingException;
+import org.opensearch.dataprepper.logstash.LogstashBaseVisitor;
+import org.opensearch.dataprepper.logstash.LogstashParser;
+import org.opensearch.dataprepper.logstash.model.LogstashConfiguration;
+import org.opensearch.dataprepper.logstash.model.LogstashPlugin;
+import org.opensearch.dataprepper.logstash.model.LogstashAttribute;
+import org.opensearch.dataprepper.logstash.model.LogstashAttributeValue;
+import org.opensearch.dataprepper.logstash.model.LogstashPluginType;
+import org.opensearch.dataprepper.logstash.model.LogstashValueType;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.LinkedHashMap;
+import java.util.stream.Collectors;
+
+/**
+ * Class to populate Logstash configuration model objects using ANTLR library classes and generated code.
+ *
+ * @since 1.2
+ */
+@SuppressWarnings("rawtypes")
+class LogstashVisitor extends LogstashBaseVisitor {
+
+    @Override
+    public Object visitConfig(LogstashParser.ConfigContext configContext) {
```

Comment:
We can mark every parameter for all methods implemented in this class as `final`